### DISCUSSION TITLE
Tile Device Trackers

### DISCUSSION THREAD

**Discussion by JAudi23** [andrew-codechimp/HA-Battery-Notes#648] [General]
Is it possible to include the Tile Device Trackers in this library?
When I attempt to add one that I have already in my HA via the Tile Integration, I am unable to pull it up.
The tracker is listed as a "device_tracker" in my HA,

**Comment by andrew-codechimp** [comment]
I don't think Tile trackers have a HA Device, which is what I require to link the battery note to, and other things that are coming soon.

There's a few devices like this which don't have the typical HA Entities under a Device structure unfortunately.
